genetic_algorithm.exs

```diff
@@ -118,16 +118,16 @@ defmodule GeneticAlgorithmAbc do
     tensor
     |> Nx.shape()
     |> elem(0)
-    |> (&Range.new(0, &1 - 1)).()
-    |> Enum.map(&if &1 < max_zeros, do: 0, else: 1)
+    |> then(&(0..(&1 - 1)))
+    |> Enum.map(&if(&1 < max_zeros, do: 0, else: 1))
     |> Enum.shuffle()
     |> Nx.tensor()
   end
 end
 
 ## Run it
 
-max_generations = 5000
+max_generations = 1000
```

Comment:
Don't change  the value here or the algorithm can't find the perfect chromosome.
Generally it takes between 3500 and 5500 to find the correct one.